bi_view_editor/models/bve_view.py

```diff
@@ -98,6 +98,11 @@ def _inverse_serialized_data(self):
         compute='_compute_users',
         store=True)
     query = fields.Text(compute='_compute_sql_query')
+    over_condition = fields.Text(
+        string="Over condition",
+        help="Condition to be inserted in the OVER part "
+             "of the ID's row_number function.",
```

Comment:
Could you add extra information regarding how to use this functionality? Or maybe some example of use?

Reply:
Added an example, generic extra information (PARTITION BY, RANGE, ...) would be way too long for an `help` attribute I think.

Reply:
Thanks. Any useful extra explanation could be added in the readme.

Reply:
Added